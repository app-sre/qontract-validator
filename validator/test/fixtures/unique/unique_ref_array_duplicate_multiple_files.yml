schemas:
  "schema-1.yml":
    properties:
      ref_array:
        type: array
        items:
          "$ref": "another-schema-1.yml"
  "another-schema-1.yml":
    properties:
      simple_field:
        type: string
graphql:
  $schema: "graphql-schema-1.yml"
  confs:
  - name: Schema_v1
    fields:
    - name: ref_array
      type: AnotherSchema_v1
      isList: true
  - name: AnotherSchema_v1
    fields:
    - name: simple_field
      type: string
      isUnique: true
  - name: Query
    fields:
    - type: Schema_v1
      datafileSchema: "schema-1.yml"
data:
  file-1-schema-1.yml:
    $schema: schema-1.yml
    ref_array:
    - simple_field: bla
  file-2-schema-1.yml:
    $schema: schema-1.yml
    ref_array:
    - simple_field: bla
resources: {}
