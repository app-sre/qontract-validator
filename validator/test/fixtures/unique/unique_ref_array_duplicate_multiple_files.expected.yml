schemas:
  "schema-1.yml":
    properties:
      ref_array:
        type: array
        items:
          "$ref": "another-schema-1.yml"
  "another-schema-1.yml":
    properties:
      simple_field:
        type: string
      __identifier: # +
        type: string
graphql:
  $schema: "graphql-schema-1.yml"
  confs:
  - name: Schema_v1
    fields:
    - name: ref_array
      type: AnotherSchema_v1
      isList: true
  - name: AnotherSchema_v1
    fields:
    - name: simple_field
      type: string
      isUnique: true
  - name: Query
    fields:
    - type: Schema_v1
      name: schema_v1
      datafileSchema: "schema-1.yml"
data:
  file-1-schema-1.yml:
    $schema: schema-1.yml
    ref_array:
    - simple_field: bla
      __identifier: "128ecf542a35ac5270a87dc740918404" # +
  file-2-schema-1.yml:
    $schema: schema-1.yml
    ref_array:
    - simple_field: bla
      __identifier: "128ecf542a35ac5270a87dc740918404" # +
resources: {}
